alembic/ddl/impl.py

```diff
@@ -136,6 +139,32 @@ def static_output(self, text: str) -> None:
         self.output_buffer.write(text + "\n\n")
         self.output_buffer.flush()
 
+    def version_table_impl(
+        self, *, version_table, version_table_schema, version_table_pk, **kw
```

Comment:
**Federico Caselli** (CaselIT) wrote:

missing typing. Should likely be:
```
 version_table: str, version_table_schema: Optional[str], version_table_pk: bool, **kw: Any
```

View this in Gerrit at https://gerrit.sqlalchemy.org/c/sqlalchemy/alembic/+/5556

Reply:
Fixed.